modules/aws/acme_certificate/variables.tf

```diff
@@ -18,7 +18,7 @@ variable "base_domain_name" {
   type        = string
 
   validation {
-    condition     = can(match("^(?!(http://|https://)).+", var.base_domain_name))
+    condition     = !can(regex("^https?://", var.base_domain_name))
```

Comment:
Will this still reject if the input has http or https?

Reply:
This is the inverse of your regex, because terraform regexes don't support lookbehinds